trigger-actions-framework/main/default/objects/DML_Finalizer__mdt/fields/Order__c.field-meta.xml

```diff
@@ -2,11 +2,11 @@
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>Order__c</fullName>
     <externalId>false</externalId>
-    <fieldManageability>DeveloperControlled</fieldManageability>
+    <fieldManageability>SubscriberControlled</fieldManageability>
     <label>Order</label>
-    <precision>18</precision>
+    <precision>6</precision>
     <required>true</required>
-    <scale>3</scale>
+    <scale>2</scale>
```

Comment:
This will truncate data for anyone who is already using all 3 decimal places.

Let's keep it at 3.

Reply:
Done